build.gradle.kts

```diff
@@ -13,8 +13,6 @@ allprojects {
     extensions.configure<SpotlessExtension> {
         kotlin {
             target("src/**/*.kt")
-            // Sources copied from AndroidX Compose
-            targetExclude("src/main/kotlin/androidx/**")
```

Comment:
Removed this bypass for `src/main/kotlin/androidx/` as I see code under this module has been changed, it's different from the original one.